code/style.css

```diff
@@ -1,7 +1,551 @@
+  font-weight: 800;
+}
+.accordion-header {
+  font-size: 18px;
+  padding: 5px;
+  margin: 2px;
+  font-weight: 500;
+  text-align: left;
+  color: black;
+  background-color: white;
+  font-family: "Playfair Display", serif;
+  cursor: pointer;
+  position: relative;
+}
+.accordion-header:hover {
+  color: #deb887;
+  cursor: pointer;
+  transition: 0.2s ease;
+}
+.accordion-header::after {
+  content: "☽";
+  padding: 10px;
+  transition: 0.2s;
+  color: #292524;
+  position: relative;
+}
+.accordion-header.active::after {
+  content: "☾";
+  padding: 10px;
+  transition: 0.2s;
+  color: #deb887;
+}
+.accordion-text {
+  border-top: 2px solid #292524;
+  font-family: "Playfair Display", serif;
+  font-size: 16px;
+  font-weight: 500;
+  margin: 0;
+  padding: 15px;
+}
+.accordion-text-container {
+  height: 0;
+  overflow: hidden;
+  transition: height 0.3s ease-in;
+}
+
+/* FOOTER */
+.footer-container {
+  background-color: #deb887;
+  background-image: linear-gradient(
+    to right,
+    #f1e1cd,
+    #f6e2c9,
+    #fce3c3,
+    #ffe4bc
+  );
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+  max-width: 100%;
+  max-height: 100px;
+  overflow: hidden;
+  padding: 20px;
+  font-family: "Sacramento", cursive;
+  font-size: 50px;
+  letter-spacing: 2px;
+}
+.footer-h2 {
+  font-size: 25px;
+  margin: 0;
+  margin-left: 5px;
+}
+.social-media {
+  display: flex;
+  flex-direction: row;
+}
+.social-icon {
+  height: 40px;
+  width: 40px;
+  margin-right: 5px;
+  cursor: pointer;
+  transition: 0.1s ease;
+  margin: 5px;
+}
+.social-icon:hover {
+  fill: whitesmoke;
+  transition: 0.3s ease;
+}
+
+/* for JS */
+.accordion-text-container.active {
+  height: 158px;
+}
+
+/* MEDIA QUERIES */
+
+@media (max-width: 900px) {
+  .rwd-image {
+    width: 100%;
+  }
+}
+@media (min-width: 901px) {
+  .rwd-image {
+    width: auto;
+  }
+}
+@media (min-width: 375px) and (max-width: 768px) {
+  .about-p {
+    font-size: 15px;
+    font-weight: 500;
+    text-align: center;
+    margin: 10px;
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+  #hero-video h1 {
+    font-size: 40px;
+  }
+}
+@media (max-width: 813px) {
+  .about-section h2 {
+    font-size: 45px;
+  }
+  #accordion {
+    margin: 15px;
+  }
+  #sign-up {
+    padding-top: 10px;
+  }
+
+  .form-sign-up h3 {
+    font-size: 50px;
+  }
+  .form-sign-up .btn {
+    width: 80%;
+  }
+  #hero-video h1 {
+    font-size: 40px;
+  }
+  .about-p {
+    font-size: 15px;
+    margin: 10px;
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+  .footer-h2 {
+    font-size: 24px;
+  }
+  li {
+    font-size: 15px;
+  }
+  #hero-video .btn,
+  .form-sign-up .btn {
+    font-size: 15px;
+  }
+  label {
+    font-size: 15px;
+  }
+  .accordion-header,
+  .accordion-text {
+    font-size: 15px;
+  }
+}
+@media (min-width: 768px) and (max-width: 1024px) {
+  #hero-video h1 {
+    font-size: 80px;
+  }
+  .about-p {
+    font-size: 30px;
+  }
```

Comment:
For a smaller device this text size is rather big - try to keep it smaller and easier to read